script.js

```diff
@@ -39,13 +39,26 @@ class App extends React.Component {
 
   }
   signIn(email, password) {
-    // sign in APIs here
-    this.setState({
-      user: {
-        email,
-        password } });
-
-
+		axios.post('http://localhost:3000',{
+			username: email,
+			password: password
+		})
+		.then((response) => {
+			console.log(response);
```

Comment:
Probably shouldn't leave console logs in code for master branch

Reply:
All console logs removed